common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -405,15 +405,15 @@ def check_button_name(self):
         """
         Determine the name for the "check" button.
 
-        Usually it is just "Check", but if this is the student's
```

Comment:
Nice work for cleanup the documentation. But usually it's better just to add an inline comment rather than changing anything edX.